ntbs-service/package.json

```diff
@@ -15,9 +15,11 @@
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
     "accessible-autocomplete": "2.0.2",
+    "array-from-polyfill": "^1.0.1",
     "axios": "0.19.2",
     "babel-loader": "^8.0.6",
     "es6-promise": "4.2.8",
+    "formdata-polyfill": "^3.0.19",
```

Comment:
Generally we should pin versions in package.json by removing the hat  - so that there's 'no' risk of a new dependence release impacting our releases (unless we manually increment versions).